sos/report/__init__.py

```diff
@@ -652,13 +652,18 @@ def _set_tunables(self):
     def _check_for_unknown_plugins(self):
         import itertools
         for plugin in itertools.chain(self.opts.only_plugins,
-                                      self.opts.skip_plugins,
                                       self.opts.enable_plugins):
             plugin_name = plugin.split(".")[0]
             if plugin_name not in self.plugin_names:
                 self.soslog.fatal('a non-existing plugin (%s) was specified '
                                   'in the command line' % (plugin_name))
                 self._exit(1)
+        for plugin in self.opts.skip_plugins:
+            if plugin not in self.plugin_names:
+                self.soslog.warning(
+                    "Requested to skip non-existing plugin '%s'" % plugin
```

Comment:
Minor comment 1: add a dot to finish the sentence?

Reply:
Our other plugin enablement warnings don't have a period, so I was matching that. I'll add a period to the other messages as well then for consistency sake ;-)